composer.json

```diff
@@ -1,9 +1,13 @@
 {
   "name": "spryker-eco/payone",
   "autoload": {
-    "psr-0": {
-      "SprykerEco": "src/",
-      "SprykerEcoTest": "tests/"
+    "psr-4": {
+      "SprykerEco\\": "src/"
+    }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "SprykerEcoTest\\": "tests/"
```

Comment:
This is not valid psr4, please check core how it is done (dev-master branch on spryker/spryker).